packages/snaps-execution-environments/src/common/validation.ts

```diff
@@ -178,6 +179,33 @@ export function assertIsOnTransactionRequestArguments(
   );
 }
 
+export const OnNameLookupRequestArgumentsStruct = object({
+  id: ChainIdStruct,
+  content: string(),
```

Comment:
Do we think `content` is the best word to choose for this payload?

Reply:
Changed to `domain`, I think that might be best since that seems to be the standard term in the industry and what we use internally.